### DISCUSSION TITLE
Changes to "$ forge remappings"

### DISCUSSION THREAD

**Discussion by 0xAa1C** [foundry-rs/foundry#822] [Ideas]
Hi everyone!

After creating a 'remappings.txt' file and adding a custom remapping I ran 'forge remappings'. I was surprised (and a bit confused) by not seeing my custom remapping in the output of that command. On top of that, I kept seeing default remappings I deleted. After reading the CLI readme I realized that this is the expected behavior, to show the default remappings. I think 'forge remappings' would be more useful if it shows all remappings currently on the project.

Seeing the default remappings (even if deleted) could be useful in some scenarios, but there should be a way to see the difference between active and deleted default remappings.

**Comment by s1monAF** [comment]
I got a different problem running `forge remappings` again after running it to create the `remappings.txt` file. Here is the error message:
```
PS C:\Users\...\test_foundry> forge remappings
Error: 
failed to extract foundry config:
foundry config error: stream did not contain valid UTF-8 in Remapping Provider
```
I'm wondering why I got this error and how this could be fixed. Thanks.
